### DISCUSSION TITLE
SOC2 Type II Audit

### DISCUSSION THREAD

**Discussion by gord-lyr** [Security-Onion-Solutions/securityonion#7889] [General] [answered]
We use the SecOnion AMI on Amazon Marketplace. As part of SOC2 Type II Audit, I'm being asked if SOS has any similar audit report or something to verify that secure development practices are followed.

Any help or direction if someone else has gone through the SOC2 Type II process??

Thanks,
Gord T.

**Comment by dougburks** [comment, marked as answer]
> secure development practices are followed.

https://docs.securityonion.net/en/2.3/security.html#product-and-supply-chain-integrity
